tests/Source/UnfittedIntegrationTests/UnfittedIntegrationTests.m

```diff
@@ -9,7 +9,10 @@
 
     methods (Test, TestTags = {'UnfittedIntegration', 'Classic', 'Perimeter', 'Circle'})
         function testPerimeterCircle(testCase, circleTests)
-            cd ../../../ % NOOOO, al segon test torna a saltar enrere
+            % Changes in directories are used to fix the current directory
+            % so that Unit Tests can pass. This should not be an issue when
+            % OldSettings disappear.
+            cd ../../../
             s.testName              = circleTests;
             s.analyticalValue       = 2*pi;
             s.meshType              = 'BOUNDARY';
```

Comment:
Hi and thanks for your contributions! Im happy to see someone is working on Swan :smile: 

I see from your comment that changing directories is a temporal workaround, but maybe you can consider using this:

https://es.mathworks.com/help/matlab/ref/matlab.unittest.fixtures.currentfolderfixture-class.html

which seems to be the official supported way of dealing with changes in the working directory. It seems this utility automatically recovers the old directory so you would save one line.

Reply:
Hi Marc! I think I've now solved it in the latest commits. Thanks a lot for reviewing!